src/content/docs/en/guides/cms/storyblok.mdx

```diff
@@ -367,24 +367,28 @@ export default defineConfig({
 
 ### Generating pages
 
-To create a route for a specific `page`, you can fetch its content directly from the Storyblok API and pass it to the `Page` component. Create an `index.astro` file in `src/pages/` to render the `home` page:
+To create a route for a specific `page`, you can fetch its content directly from the Storyblok API and pass it to the `StoryblokComponent` component.  Remember to make sure you have added the `Page` component to your astro.config.mjs.
```

Comment:
```suggestion
To create a route for a specific `page`, you can fetch its content directly from the Storyblok API and pass it to the `StoryblokComponent` component.
```
This reverts a change I made that in retrospect was problematic, so I'm happy with this PR! I don't think we need this line, though, as registering the page component happened in the previous step.

Reply:
Looks good!